functests/10-test-template-validator-placement.sh

```diff
@@ -27,6 +29,8 @@ if [ "$DEPLOYMENT_FOUND" == "false" ]; then
   exit 1
 fi
 
+oc get -n ${TEST_NS} deploy virt-template-validator -ojson | jq '.spec.template.spec'
+
```

Comment:
Why is this command here? Is it for logging?

Reply:
Yes, `test-runner.sh` hides it during `make functests`, and if the test fails, it will print it